changelog.d/14619.doc

```diff
@@ -0,0 +1 @@
+Fix incorrect naming of `push.enabled` configuration option in documentation.
```

Comment:
Since we haven't released #14551 yet, can you copy its changelog text here? That way the two PRs will be amalgamated when we do the release.

Reply:
Ah, that was the trick to it! Sure.